spec/components/info_metric_spec.rb

```diff
@@ -23,8 +23,9 @@
   end
 
   it "does not render if figure is not supplied" do
```

Comment:
As above - make sure your test description matches exactly what you expect to see.